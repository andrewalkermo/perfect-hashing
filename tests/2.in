i
16
joao
20
i
18
maria
30
i
27
carlos
15
i
29
ana
18
i
11
Nat
15
i
1
Nat
15
i
43
Nat 
25
i
45
Nat 
28
i
89
Nat
23
i
77
Nat
44
i
55
Nat
12
p
r
55
p
r
16
p
r
18
p
r
27
p
r
29
p
r
11
p
r
1
p
r
77
p
r
55
p
r
89
p
r
45
p
r
43
p
i
16
joao
20
i
18
maria
30
i
27
carlos
15
i
29
ana
18
i
11
Nat
15
i
1
Nat
15
i
43
Nat 
25
i
45
Nat 
28
i
89
Nat
23
i
77
Nat
44
i
55
Nat
12
p
e