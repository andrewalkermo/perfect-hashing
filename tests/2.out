estrutura de hashing perfeito criada
chave: 72
nome nove
82
chave nao encontrada: 55
hashing perfeito: primeiro nivel
tamanho da tabela: 9
parametro a: 3
parametro b: 42
numero primo: 101
0: 10
2: 60 72 75
5: 70
7: 40 52 22 37
hashing perfeito: segundo nivel - indice: 7
tamanho da tabela: 16
parametro a: 23
parametro b: 88
numero primo: 101
3: 40
8: 52
9: 22
14: 37
10100